source/amazon/services/supported-services/ecr-image-scanning.rst

```diff
@@ -29,6 +29,149 @@ AWS configuration
 
 AWS provides a `template <https://github.com/aws-samples/ecr-image-scan-findings-logger/blob/main/Template-ECR-SFL.yml>`_ for creating a stack in CloudFormation that loads the image scan findings from Amazon ECR in CloudWatch using an AWS Lambda function.
 
+Policy configuration
+^^^^^^^^^^^^^^^^^^^^
+.. include:: /_templates/cloud/amazon/create_policy.rst
+
+Template specific permissions
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+IAM 
+"""
+
+.. warning::
+      The permissions inside the ``RoleCreator`` and ``PassRole`` sections are necessary in order to create/delete the stack based on the named template and must be bound to the described specific resources due to overly permissive actions.
+
+.. code-block:: json
+  
+   {
+      "Sid": "RoleCreator",
+      "Effect": "Allow",
+      "Action": [
+         "iam:CreateRole",
+         "iam:PutRolePolicy",
+         "iam:AttachRolePolicy",
+         "iam:DeleteRolePolicy",
+         "iam:DeleteRole",
+         "iam:GetRole",
+         "iam:GetRolePolicy",
+         "iam:PassRole"
+      ],
+      "Resource": "arn:aws:iam::user-id:role/*"
```

Comment:
```suggestion
      "Resource": "arn:aws:iam::<user-id>:role/*"
```
We need to be clear when some parts of the Resource definition are variables that must be configured by the user. We should use `<` or some other way to draw the user's attention to it. Apply this change to the rest of the code-blocks when applicable.

Besides, I think what you really wanted to indicate is `account ID` instead of `user-id`, but I may be wrong. Check this and update accordingly.